test/pkg/environment/common/expectations.go

```diff
@@ -458,6 +486,26 @@ func (env *Environment) ConsistentlyExpectNodeCount(comparator string, count int
 	return lo.ToSlicePtr(nodeList.Items)
 }
 
+func (env *Environment) ConsistentlyExpectNoDisruptions(nodeCount int, duration string) {
```

Comment:
Thoughts on hardcoded duration?

Reply:
I could go either way. Maybe overfactoring too early, but I also think that you naturally ask the "how long" question when it comes to `Consistently` blocks. So, I could realistically see making it a standard that when you add a `Consistently` block, you pass in a duration so that the caller is clear on how long that you are checking for at the maximum

Reply:
I personally think it's best to do it the two function route. Most calls just use a default, but you can use a configured function if you like. Up to you if you think that's overfactoring.

Reply:
I think I'm happy to leave it the way that it is right now.